lib/block.mli

```diff
@@ -14,6 +14,16 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
+(** Code blocks headers. *)
+
+module Header : sig
+  type t = Shell | OCaml | Other of string
```

Comment:
The `OCaml` constructor is also in the `value` type below, which is more important, I think.
This is not really an issue but can be slightly confusing.